meta-sokol-flex-distro/recipes-core/images/flex-initramfs-image.bb

```diff
@@ -29,7 +29,7 @@ BAD_RECOMMENDATIONS += "busybox-syslog"
 # We don't need selinux labels in initramfs
 IMAGE_PREPROCESS_COMMAND:remove = "selinux_set_labels ;"
 
-COMPATIBLE_HOST:sokol-flex = "(arm|aarch64|i.86|x86_64).*-linux"
+COMPATIBLE_HOST:sokol-flex = "(arm|aarch64|i.86|x86_64|riscv64).*-linux"
```

Comment:
@kergoth @abelal @ahsanhussain  is here any benefit setting COMPATIABLE_HOST?

Reply:
@kergoth can you remove this in your PR

Reply:
Done